survey-runner.tf

```diff
@@ -159,28 +159,125 @@ module "survey-launcher-for-ecs" {
   EOF
 }
 
-module "author" {
-  source                       = "github.com/ONSdigital/eq-author-deploy"
-  env                          = "${var.env}"
-  aws_access_key               = "${var.aws_access_key}"
-  aws_secret_key               = "${var.aws_secret_key}"
-  dns_zone_name                = "${var.dns_zone_name}"
-  ecs_cluster_name             = "${module.eq-ecs.ecs_cluster_name}"
-  aws_alb_listener_arn         = "${module.eq-ecs.aws_alb_listener_arn}"
-  application_cidrs            = "${concat(var.ecs_application_cidrs, var.application_cidrs)}"
-  docker_registry              = "${var.author_registry}"
-  author_tag                   = "${var.author_tag}"
-  author_api_tag               = "${var.author_api_tag}"
-  publisher_tag                = "${var.publisher_tag}"
-  survey_launcher_url          = "${module.survey-launcher-for-ecs.service_address}"
-  enable_auth                  = "${var.author_enable_auth}"
-  firebase_project_id          = "${var.author_firebase_project_id}"
-  firebase_api_key             = "${var.author_firebase_api_key}"
-  firebase_messaging_sender_id = "${var.author_firebase_messaging_sender_id}"
-  schema_validator_url         = "${module.schema-validator.service_address}"
-  author_min_tasks             = "${var.author_min_tasks}"
-  author_api_min_tasks         = "${var.author_api_min_tasks}"
-  publisher_min_tasks          = "${var.publisher_min_tasks}"
+module "author-for-ecs" {
```

Comment:
I don't think we need to specify `-for-ecs` that was used for launcher to differentiate between the two of them